test/Microsoft.AspNetCore.Server.KestrelTests/HttpsConnectionFilterTests.cs

```diff
@@ -301,5 +312,59 @@ public class HttpsConnectionFilterTests
 #endif
             }
         }
+
+        // https://github.com/aspnet/KestrelHttpServer/issues/240
+        // This test currently fails on mono because of an issue with SslStream.
```

Comment:
Eh, I should have deleted that comment. It came along when I copied another test. This new test works fine on Mono.